src/ansys/geometry/core/designer/design.py

```diff
@@ -679,6 +688,45 @@ def add_beam_circular_profile(
 
         return self._beam_profiles[profile.name]
 
+    @protect_grpc
+    @min_backend_version(25, 1, 0)
+    def get_all_parameters(self) -> list[Parameter]:
+        """Get parameters for the design.
+
+        Returns
+        -------
+        list[Parameter]
+            List of parameters for the design.
+        """
+        response = self._parameters_stub.GetAll(GetAllRequest())
+        return [Parameter._from_proto(dimension) for dimension in response.driving_dimensions]
+
+    @protect_grpc
+    @check_input_types
+    @min_backend_version(25, 1, 0)
+    def set_parameter(self, dimension: Parameter) -> ParameterUpdateStatus:
+        """Update a parameter of the design.
```

Comment:
```suggestion
        """Set or update a parameter of the design.
```
It is best to use the verb in the method name--but maybe here it we could use "Set or update" because of the value returned?